third-party-podspecs/libevent.podspec

```diff
@@ -0,0 +1,261 @@
+
+Pod::Spec.new do |s|
+
+  s.name         = "libevent"
```

Comment:
so yeah, I'm not sure what pulled this in, but let's find a way to not depend on it

Reply:
removed it